tinker/templates/faculty-bios/home.html

```diff
@@ -83,6 +83,7 @@
             <div class="col-md-12">
                 {% if show_create %}
                     <a href="{{ url_for('FacultyBiosView:new') }}" class="btn btn-primary">Create New Bio</a>
+                    <a href="{{ url_for('FacultyBiosView:get_faculty_bio_csv') }}" class="btn btn-primary">Export csv file</a>
```

Comment:
You should have show/hide permissions for this. Only those that can see all faculty bios should be able to export csv

Reply:
Added permission check.